DivanteLtd/storefront-ui
# [WIP] Testing environment for Vue 3

# Scope of work

- prepared environment for testing
## How to test components
- in new folder `vue3` you can find installed Vue CLI project when testing begins 
- in `packages/vue/index.js` are listed all imports of components. 
First, you should comment all imports besides the one which you want to test
- in `App.vue` add an import of the component you want to test
- use `yarn link` in `packages/vue` after that in folder `vue3` use `yarn link "@storefront-ui/vue` to link a current version of our `@storefront-ui/vue` package and to be updated after making changes in our components
- in `vue3` use `yarn serve` to test, if you noticed any problems, bugs - write them in https://github.com/DivanteLtd/storefront-ui/issues/966
- currently, to work with Vue2, we should import `CompositionApi` if it's needed to test some components
https://github.com/vuejs/composition-api
`import CompositionApi from "@vue/composition-api";`
`Vue.use(CompositionApi);`
after that you can import method to use like this:
`import { computed } from "@vue/composition-api";`
